nixos/modules/services/web-apps/jitsi-meet.nix

```diff
@@ -189,6 +189,19 @@ in
     };
 
     secureDomain.enable = mkEnableOption (lib.mdDoc "Authenticated room creation");
+
+    updateMucs = mkOption {
+      type = with types; attrsOf anything;
+      default = { };
+      description = "This is a very hacky work around setting and only added because prosody muc is a list.";
```

Comment:
This descriptions tells me nothing at all about how this option is expected to be used or what it does. My understanding is that this is a way to inject extra Prosody settings for a MUC, which is difficult to do otherwise due to the combination of 1. the Prosody module not using an attrset for its instances; 2. the Jitsi module being in charge of configuring Prosody itself?

1. Have you considered refactoring Prosody to use an attrset like what your `updateMucs` option uses as its API? It should even be doable to keep compatibility with the old list format by auto-transforming the list into an attrset using v.domain.
2. If this is not possible, please improve the naming and documentation here. How it's implemented should be of no concern to a user of the option and has (IMHO) no place in the documentation.